content/_index.md

```diff
@@ -24,18 +24,29 @@ In short, it would let anyone easily answer the question, "does this product con
 {{< /home/section >}}
 
 {{< home/section title="How?" >}}
+**How does [GitBOM](glossary/gitbom/) improve software identification and vulnerability management?**
```

Comment:
You might try doing this as a header (like # or ## or ###) instead of just bold, as it would make it linkable :)

Reply:
done :)